docs/reference/contributing/target/flash.md

```diff
@@ -1,8 +1,10 @@
-### Flash
+### Flash and Bootloader
```

Comment:
Please lower case "bootloader" for consistent sentence case.

Reply:
For future endeavors: do we use sentence case in titles?

Reply:
Yes, we use sentence case in titles. I think that's in the style guide, but we may need to be better about enforcing it.